docs/spec/channel.md

```diff
@@ -1,13 +1,13 @@
-# Channel Spec
+# Knative Channel Specification
 
 ## Background
 
-Starting with Version 0.7.0 all the different Channel CRDs (e.g.
-`InMemoryChannel` or `KafkaChannel`) are living in the
-`messaging.knative.dev/v1alpha1` API Group.
+The Knative Eventing project has one generic `Channel` CRD and might ship
+different Channel CRDs implementations (e.g.`InMemoryChannel`) inside of
+in the `messaging.knative.dev/v1beta1` API Group.
```

Comment:
Should we keep history of changes such as API group was changed to messaging.knative.dev/v1beta1 in version 0.x.y ?